lib/github-events.js

```diff
@@ -16,16 +10,15 @@ module.exports = (app) => {
       return res.end()
     }
 
-    const signature = req.headers['x-hub-signature']
-    const data = req.body
-    data.action = data.action ? event + '.' + data.action : event
-
-    if (!signature || signature !== sign(secret, req.raw)) {
+    if (!githubSecret.isValid(req)) {
```

Comment:
Extracted into its own local module to make it easier to turn off the verification while running tests.